Formula/adb-enhanced.rb

```diff
@@ -0,0 +1,20 @@
+class AdbEnhanced < Formula
+  include Language::Python::Virtualenv
+
+  desc "Swiss-army knife for Android testing and development"
+  homepage "https://ashishb.net/tech/introducing-adb-enhanced-a-swiss-army-knife-for-android-development/"
+  url "https://github.com/ashishb/adb-enhanced/tree/2.5.18"
+  # version "2.5.18"
```

Comment:
need build from source

Reply:
Changed to "https://files.pythonhosted.org/packages/78/ac/7e5f1e218704183106e332e720e421ebbed6f90a9a10946560faf1e33e74/adb-enhanced-2.5.18.tar.gz"